src/time/src/mcp_server_time/server.py

```
@@ -40,10 +40,30 @@ def get_local_tz(local_tz_override: str | None = None) -> ZoneInfo:
         return ZoneInfo(local_tz_override)
 
     # Get local timezone from datetime.now()
-    tzinfo = datetime.now().astimezone(tz=None).tzinfo
+    local_dt = datetime.now().astimezone(tz=None)
+    tzinfo = local_dt.tzinfo
+    
     if tzinfo is not None:
-        return ZoneInfo(str(tzinfo))
-    raise McpError("Could not determine local timezone - tzinfo is None")
+        try:
+            # Try to get the IANA timezone name directly
+            return ZoneInfo(str(tzinfo))
+        except Exception:
```

I don't think you can just swallow an `Exception` here and assume you can just parse something other than `tzinfo` a different way...